### PR TITLE
rtmp-services: Add NFHS Network

### PR DESCRIPTION
**Pull request by migillett**
### Description
Added rtmp service for the [NFHS Network](https://www.nfhsnetwork.com/).

We modified the `package.json` and `services.json` files inside of the `rtmp-services` plugin to include the primary RTMP endpoint for the NFHS Network. The changes to the plugin also include our supported resolutions and maximum frame rate.

### Motivation and Context
Our primary motivation for this was to simplify the process of getting our customers streaming to our service and prevent potential user errors when re-typing or pasting the RTMP URL.

### How Has This Been Tested?
- JSON Validation
- Built using cmake and compiled using XCode on a 2021 Macbook Pro (Apple M1 Pro) running MacOS Sonoma 14.4.1
- Performed a test stream to our service using the new NFHS Network service dropdown option
- Verified RTMP stream displayed on the service

### Types of changes
- New feature (non-breaking change which adds functionality)

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by norihiro** [comment]
The first word should be capitalized. `add` --> `Add`.

**Comment by migillett** [comment]
@norihiro Apologies, updated name of PR to `rtmp-services: Add NFHS Network`

**Review by Fenrirthviti — APPROVED** [review]
Approved from an eligibility standpoint.

**Comment by norihiro** [comment]
> @norihiro Apologies, updated name of PR to `rtmp-services: Add NFHS Network`

The commit message has not been fixed.
You need to run `git commit --amend` and edit the commit message. Then, you need to force-push the commit to GitHub.

**Comment by migillett** [comment]
@norihiro And fixed! Apologies, first time contributing to the project.

**Comment by RytoEX** [comment]
Unfortunately, with the merge of #11080, this will need to be rebased to correctly increment the versions in package.json.

**Comment by migillett** [comment]
I fixed the merge conflict and bumped the rtmp services version from 260 (master) to 261. @RytoEX

**Review by RytoEX — CHANGES_REQUESTED** [review]
Do not use merge commits to fast forward history. Use `git rebase`.

**Comment by migillett** [comment]
Sounds good, I'm on it @RytoEX. I'll ping you when it's ready.

**Comment by migillett** [comment]
@RytoEX Ok, I think that did it. Rebased to remote master and squashed all of my commits into one. Also made sure that the commit message was correct like specified earlier.

File: plugins/rtmp-services/data/package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+    "version": 261,
     "files": [
         {
             "name": "services.json",
-            "version": 260
+            "version": 261
         }
     ]
-}
+}
```
Comment (RytoEX) [inline plugins/rtmp-services/data/package.json:11]: Revert this change. Always end the file with a newline.

**Review by RytoEX — CHANGES_REQUESTED** [review]

File: plugins/rtmp-services/data/package.json (hunk 1 above)
Comment (migillett) [reply plugins/rtmp-services/data/package.json]: Fixed!

**Review by RytoEX — APPROVED** [review]
